src/tests/queryFilters/auditEventsRecorded/auditEventsRecorded.test.js

```diff
@@ -0,0 +1,90 @@
+const supertest = require('supertest');
+
+const {app} = require('../../../app');
+// provider file
+const auditEventResource = require('./fixtures/auditEvents.json');
+
+// expected
+const expectedAuditEventResource = require('./fixtures/expectedAuditEvents.json');
+
+const request = supertest(app);
+const {commonBeforeEach, commonAfterEach, getHeaders} = require('../../common');
+
+describe('AuditEventRecordedTests', () => {
+    beforeEach(async () => {
+        await commonBeforeEach();
+    });
+
+    afterEach(async () => {
+        await commonAfterEach();
+    });
+
+    describe('AuditEvent Recorded Tests', () => {
+        test('search by recorded works', async () => {
```

Comment:
For test cases in the future, should we go with the AAA (arrange, act, assert) commenting?

Reply:
yes, we should start doing that